capitalone/rubicon-ml
# deprecate `ui` submodule

closes: #165 
---

## What
  * `ui.Dashboard` now raises a `NotImplementedError` that directs users to `viz.Dashboard`
  * `viz.Dashboard` now has default widgets that replicate the old `ui.Dashboard`
  * the CLI leverages `viz.Dashboard`
  * the existing notebooks leverage `viz.Dashboard`
  * the `ui` submodule is emptied except for `ui.Dashboard` with the deprecation message, which will be removed eventually

## How to Test
  * validate that the updated notebooks run as expected
  * validate that you can launch a dashboard from the CLI
    * [these instructions](https://capitalone.github.io/rubicon-ml/dashboard.html#cli) should still be applicable
